tools/serverpod_cli/test/analyzer/models/stateful_analyzer/model_validation/field/field_restricted_names_test.dart

```diff
@@ -94,7 +94,7 @@ void main() {
     'findSingleRow',
     'find',
     'setColumn',
-    'allToJson',
+    'toJson',
```

Comment:
Chore: update this in the same way as I commented on the restricted field implementation.